ca.vlacroix.Tally.json

```diff
@@ -0,0 +1,50 @@
+{
+	"app-id": "ca.vlacroix.Tally",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "47",
+	"sdk": "org.gnome.Sdk",
+	"command": "tally",
+	"finish-args": [
+		"--device=dri",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [{
+		"name": "lua",
+		"buildsystem": "simple",
+		"build-commands": [
+			"make",
+			"make install INSTALL_TOP=/app"
+		],
+		"sources": [{
+			"type": "archive",
+			"url": "https://lua.org/ftp/lua-5.4.7.tar.gz",
+			"sha256": "9fbf5e28ef86c69858f6d3d34eccc32e911c1a28b4120ff3e84aaa70cfbf1e30"
+		}]
```

Comment:
is it possible to use this through the shared modules ? https://github.com/flathub/shared-modules/tree/master/lua5.4

I can bump the version if you need 5.4.7 specifically, it has 5.4.6

Reply:
I did not specifically need 5.4.7, that was just the latest release and so it's what I chose when I first started building this with Flatpak. I'm quite new to using Flatpak and so I was unfamiliar with these concepts until you suggested them. I've now made the fixes you suggested.